test/exercises/wavelet/wavelet.chpl

```diff
@@ -0,0 +1,66 @@
+use ecgIO;
+import Math.log2;
+
+config const nLevels = 3;
+config param plotResults = false;
```

Comment:
I'd make this a config const as well — it's not a case that's going to benefit heavily from compile-time optimization, and it's more convenient to be able to change the flag from run-to-run rather than compile-to-compile.  (and, the default can still be changed at compile-time if desired).  Then I'd combine these two declarations into one to avoid the repeated `config const`.